[SDMEXT-764] Exception messages on UI

rishikunnath2747 opened the pull request:
This PR modifies the exception messages to make them more intuitive, and throws the exact error message that is received from DI in case the error is neither duplicate nor virus. Exception messages are now visible on the UI after the latest UI5 update.

[Link to JIRA BL](https://jira.tools.sap/browse/SDMEXT-764)

rishikunnath2747 committed:
Fix + UT

This commit changes the error message for Creating an attachment and modifies the respective UT

rishikunnath2747 committed:
Fix

Removing comments

rishikunnath2747 committed:
Updating error messages

rishikunnath2747 committed:
Update error messages

rishikunnath2747 committed:
Updating error message

This commit changes the error response to whatever DI throws in case of an error other than duplicate or virus

rishikunnath2747 committed:
UTs

This commit adds UTs. The code has been tested and validated on cloud

rashmiangadi11 commented on sdm/src/main/java/com/sap/cds/sdm/service/handler/SDMAttachmentsServiceHandler.java:
```diff
@@ -90,12 +90,13 @@ public void createAttachment(AttachmentCreateEventContext context) throws IOExce
 
           if (createResult.get("status") == "duplicate") {
-            throw new ServiceException("The following file already exists and cannot be uploaded");
+            throw new ServiceException(filename + " already exists.");
```
can we put this in constants file

rishikunnath2747 replied:
Fixed

rishikunnath2747 committed:
Error Messages

This PR moves all the error messages to the constants file so that they can be reused
